apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktOppgaveService.kt

```diff
@@ -55,16 +65,102 @@ class AktivitetspliktOppgaveService(
+            if (ferdigstilt12mndOppgave.isNotEmpty()) {
+                throw Har12MndVurderingFerdigstilt(
+                    "Kan ikke opprette 6 mnd vurdering, finnes en ferdigstilt eller under arbeids oppgave for 12 mnd vurdering",
+                )
+            }
+        }
+        if (request.type == VurderingType.TOLV_MAANEDER) {
+            val oppfoelging6mnd =
+                oppgaveService.hentOppgaverForSakAvType(
+                    sakId,
+                    listOf(OppgaveType.AKTIVITETSPLIKT),
+                )
+            if (oppfoelging6mnd.any { it.erUnderBehandling() }) {
+                throw KanIkkeopprette12mndOppaveOm6MndErUnderbehandling("Kan ikke opprette 12 mnd mens en 6 mnd er under behandling. ")
+            }
+            val ferdigstilt6mndOppgave = oppfoelging6mnd.filter { it.erFerdigstilt() }
+            if (ferdigstilt6mndOppgave.isEmpty()) {
+                throw MaaHa6mndVurderingForAaOpprette12mnd("Kan ikke opprette 12 mnd vurdering, uten en ferdigstilt 6 mnd vurdering")
+            }
+        }
+
+        val oppgaveType =
+            when (request.type) {
+                VurderingType.SEKS_MAANEDER -> OppgaveType.AKTIVITETSPLIKT
+                VurderingType.TOLV_MAANEDER -> OppgaveType.AKTIVITETSPLIKT_12MND
+            }
+
+        val oppgaverForSak = oppgaveService.hentOppgaverForSakAvType(sakId, listOf(oppgaveType))
+        val harOppfoelgingsOppgaveUnderbehandling =
+            oppgaverForSak.filter {
+                it.erUnderBehandling() ||
+                    it.status == no.nav.etterlatte.libs.common.oppgave.Status.AVBRUTT
+            }
+
+        if (harOppfoelgingsOppgaveUnderbehandling.isNotEmpty()) {
+            throw HarOppfoelgingsOppgaveUnderbehandling(
+                "Det finnes allerede en tilsvarende oppgave som ikke er ferdigbehandlet for denne saken. Sakid=$sakId",
+            )
+        }
+
+        val sisteIverksatte =
+            behandlingService.hentSisteIverksatte(sakId)
+                ?: throw ManglerIverksattBehandling("Har ingen iverksatt behandling for sak. Sakid=$sakId")
+
+        // Dette er de samme som beskrivelse i de korresponderende jobbtypene OMS_DOED_6/12MND
+        val merknad =
+            when (request.type) {
+                VurderingType.SEKS_MAANEDER -> "Vurdering av aktivitetsplikt ved 6 måneder"
+                VurderingType.TOLV_MAANEDER -> "Vurdering av aktivitetsplikt ved 12 måneder"
+            }
+
+        val opprettetOppfoelgingsoppgave =
+            oppgaveService.opprettOppgave(
+                sakId = sakId,
+                referanse = sisteIverksatte.id.toString(),
+                kilde = OppgaveKilde.SAKSBEHANDLER,
+                type = oppgaveType,
+                merknad = merknad,
+                frist =
+                    LocalDate
+                        .now()
+                        .plusMonths(2)
```

Comment:
```suggestion
                        .plusMonths(1)
```
siden 1 er standard.

Reply:
fixd

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/6995/commits/e1f010f673856cb58557e862fcb5d2a4ba6af64c